nf_core/modules/bump_versions.py

```diff
@@ -0,0 +1,317 @@
+                "name": "all_modules",
+                "message": "Bump versions for all modules or a single named module?",
+                "choices": ["All modules", "Named module"],
+            }
+            answer = questionary.unsafe_prompt([question], style=nf_core.utils.nfcore_question_style)
+            if answer["all_modules"] == "All modules":
+                all_modules = True
+            else:
+                module = questionary.autocomplete(
+                    "Tool name:",
+                    choices=[m.module_name for m in nfcore_modules],
+                    style=nf_core.utils.nfcore_question_style,
+                ).ask()
+
+                # Only lint the given module
+        if module:
+            if all_modules:
+                raise nf_core.modules.module_utils.ModuleException(
+                    "You cannot specify a tool and request all tools to be bumped."
+                )
+            nfcore_modules = [m for m in nfcore_modules if m.module_name == module]
+            if len(nfcore_modules) == 0:
+                raise nf_core.modules.module_utils.ModuleException(f"Could not find the specified module: '{module}'")
+
+        progress_bar = rich.progress.Progress(
+            "[bold blue]{task.description}",
+            rich.progress.BarColumn(bar_width=None),
+            "[magenta]{task.completed} of {task.total}[reset] » [bold yellow]{task.fields[test_name]}",
+            transient=True,
+        )
+        with progress_bar:
+            bump_progress = progress_bar.add_task(
+                "Bumping nf-core modules versions", total=len(nfcore_modules), test_name=nfcore_modules[0].module_name
+            )
+            for mod in nfcore_modules:
+                progress_bar.update(bump_progress, advance=1, test_name=mod.module_name)
+                self.bump_module_version(mod)
+
+        self._print_results()
+
+    def bump_module_version(self, module: NFCoreModule):
+        """
+        Bump the bioconda and container version of a single NFCoreModule
+
+        Args:
+            module: NFCoreModule
+        """
+        # Extract bioconda version from `main.nf`
+        bioconda_packages = self.get_bioconda_version(module)
+
+        # If multiple versions - don't update! (can't update mulled containers)
+        if not bioconda_packages or len(bioconda_packages) > 1:
+            return (f"Ignoring mulled container {module.main_nf}", module.module_name)
+
+        # Check for correct version and newer versions
+        bioconda_tool_name = bioconda_packages[0].split("=")[0].replace("bioconda::", "").strip("'").strip('"')
+        bp = bioconda_packages[0]
+        bp = bp.strip("'").strip('"')
+        try:
+            bioconda_version = bp.split("=")[1]
+            response = nf_core.utils.anaconda_package(bp)
+        except LookupError as e:
+            self.failed.append((f"Conda version not specified correctly: {module.main_nf}", module.module_name))
+            return
```

Comment:
```suggestion
            return False
```
I'm thinking it might be useful, at least to someone reading the code, of having some sort of exit status for the function, i.e True/False.

Reply:
Yeah you're right, I will improve the exit behavior a bit!